modules/system/twig/Extension.php

```diff
@@ -1,5 +1,6 @@
 <?php namespace System\Twig;
 
+use Twig\TwigFunction as TwigSimpleFunction;
```

Comment:
Please put this next to the other twig import

Reply:
My bad, it was auto imported at the top lol. Fixed in https://github.com/wintercms/winter/pull/428/commits/26ecdb002e576d5321374c7ae073579054dcf5d9